### PR TITLE
Tell the CUPS server about the properties we're interested in.

### LINKED ISSUE #6: Printer is detected and shown, but then "currently not availble" in print dialog
Like the title says. In the actual print dialog, when selecting the printer, it says the printer is currently not available. No obvious indications as of why.  (Android 6.0.1 to CUPS/Debian)

### PR DESCRIPTION
**Pull request by maniac103**
If we don't do that, it looks like the server sends only a very limited
subset of all attributes.

Should fix #6.

### CODE REVIEW

**Comment by BenoitDuffez** [comment]
Thanks for all your PRs.
Do you think I should publish to the Play Store, or does this require more testing?

**Comment by maniac103** [comment]
That probably depends on the testing possibilities you have. #18 obviously doesn't need any testing outside of compiling, and I'm fairly confident in #16 as well due to the spec reference and the values my printer sends. The only one I'm not entirely sure about is this one (#17): while I can't see a problem in specifically asking the server for the attributes we're interested in, from my understanding of the IPP spec it shouldn't be *required* either, as when we don't specify the list of requested attributes, it should react as if 'all' was passed. As I haven't seen a strict requirement that it must return all attributes if 'all' is passed, it's probably an implementation detail what exactly is returned in that case.
Long story short: if you can easily do more testing than just go ahead with that as my testing base is only one combination of server (Ubuntu 15.10 with the cups package included in it) and printer (HP LaserJet 1018), if you can't easily test much either, I personally would release this code as my confidence in it is fairly high.
